src/utils/todo/task-filing/helper.ts

```diff
@@ -24,13 +24,13 @@ export const filingTypeToName = (
     case FilingTypes.AGM_LOCATION_CHANGE: return FilingNames.AGM_LOCATION_CHANGE
     case FilingTypes.ALTERATION: return FilingNames.ALTERATION
     case FilingTypes.AMALGAMATION_APPLICATION:
-      if (subType === FilingSubTypeE.AMALGAMATION_HORIZONTAL) {
+      if (subType === AmalgamationTypes.HORIZONTAL) {
         return `${FilingNames.AMALGAMATION_APPLICATION} Short-form (Horizontal)`
       }
-      if (subType === FilingSubTypeE.AMALGAMATION_REGULAR) {
+      if (subType === AmalgamationTypes.REGULAR) {
         return `${FilingNames.AMALGAMATION_APPLICATION} (Regular)`
       }
-      if (subType === FilingSubTypeE.AMALGAMATION_VERTICAL) {
+      if (subType === AmalgamationTypes.VERTICAL) {
```

Comment:
AmalgamationTypes exists within FilingSubTypeE, but we specify AmalgamationTypes throughout the code for the amalgamation filing